lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -3998,7 +3998,7 @@ namespace Js
             OUTPUT_VERBOSE_TRACE(Js::DebuggerPhase, _u("DebugProfileProbeThunk: calling function: %s isWrapperRegistered=%d useDebugWrapper=%d\n"),
                 function->GetFunctionInfo()->HasBody() ? function->GetFunctionBody()->GetDisplayName() : _u("built-in/library"), AutoRegisterIgnoreExceptionWrapper::IsRegistered(scriptContext->GetThreadContext()), useDebugWrapper);
 
-            if (scriptContext->IsScriptContextInDebugMode())
+            if (scriptContext->IsScriptContextInDebugRecordingMode())
```

Comment:
Still having problems with type visibility. DebugContext is not visible outside of ScriptContext.cpp where I would need it in order to call the methods. The method is not necessary here, but it is necessary to call outside of ScriptContext. I'll keep trying to refactor to reduce the number of methods needed but I think a couple will be needed.

---
In reply to: [129987071](https://github.com/Microsoft/ChakraCore/pull/3447#discussion_r129987071) [](ancestors = 129987071)